public/css/main.css

```diff
@@ -0,0 +1,62 @@
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    direction: ltr;
+    font-size: 100%;
+}
+/*start style of footer section*/
+.footer {
+  background-color: #191919;
+  height: 121px;
+  bottom: 0;
+}
+.footer__container{
+  display: flex;
```

Comment:
The space in between solution for the footer here only works within a small range of screen sizes,  Another solution that would work for a larger range of screen sizes  (still not perfect, a stretch goal we should make is adapting to mobile but strech)

Add  justify-content: space-between;  to here .footer__container
so you can remove the margin left here but you'd need ta add a margin-right ie 6% or vw
![image](https://user-images.githubusercontent.com/39469813/66469405-b03c5180-ea7f-11e9-97b5-2811f81a89ed.png)